app/components/Common/Markers/index.tsx

```diff
@@ -48,35 +77,83 @@ interface PointAnnotationMarkerProps {
   geoJSON: geoJSONType;
   i: number;
   alphabets: string[];
+  setCoordinateModalShow: React.Dispatch<React.SetStateAction<boolean>>;
+  setCoordinateIndex: React.Dispatch<React.SetStateAction<number | null | undefined>>;
+  onPressMarker: (isSampleTree: boolean, coordinate: []) => void;
+  setIsSampleTree: React.Dispatch<React.SetStateAction<boolean | null>>;
+  locateTree: string;
 }
 
 const PointAnnotationMarker = ({
   markers,
   geoJSON,
   i,
   alphabets,
+  setCoordinateModalShow,
+  setCoordinateIndex,
+  onPressMarker,
+  setIsSampleTree,
+  locateTree,
 }: PointAnnotationMarkerProps): JSX.Element[] => {
-  const annotationRefList = useRef<MapboxGL.PointAnnotation[] | null>([]);
+  const [activeAnnotationIndex, setActiveAnnotationIndex] = useState<number>(-1);
+  const [previousActiveAnnotationIndex, setPreviousActiveAnnotationIndex] = useState<number>(-1);
+  const [backgroundColor, setBackgroundColor] = useState<string>('blue');
+  const [coordinates, setCoordinates] = useState([]);
 
+  const annotationRefList = useRef<MapboxGL.PointAnnotation[] | null>([]);
+  const calloutRefList = useRef<MapboxGL.Callout[] | null>([]);
+  let scaleIn;
+  let scaleOut;
   let onePolygon = geoJSON.features[i];
 
   useEffect(() => {
     annotationRefList.current = annotationRefList.current.slice(
       0,
       onePolygon.geometry.coordinates.length,
     );
+    calloutRefList.current = calloutRefList.current.slice(
+      0,
+      onePolygon.geometry.coordinates.length,
+    );
   }, [onePolygon.geometry.coordinates]);
 
+  const onAnnotationSelected = (activeIndex, feature) => {
+    if (activeAnnotationIndex === activeIndex) {
+      return;
+    }
+
+    scaleIn = new Animated.Value(0.6);
+    Animated.timing(scaleIn, { toValue: 1.0, duration: 200 }).start();
+    // setState({ activeAnnotationIndex: activeIndex });
+    setActiveAnnotationIndex(activeIndex);
+
+    if (previousActiveAnnotationIndex !== -1) {
+      // this._map.moveTo(feature.geometry.coordinates, 500);
+    }
+  };
+
   if (onePolygon.geometry.type === 'Point') {
     markers.push(
       <MapboxGL.PointAnnotation
         key={`${i}-point`}
         id={`${i}-point`}
+        title="Test"
```

Comment:
The test "Test" does not show up in the UI, does it?

Reply:
No. it doesn't